.github/actions/rust/action.yaml

```diff
@@ -17,7 +17,6 @@ inputs:
     description: Enable Rust cache
     required: false
     default: "true"
-
```

Comment:
Why did you remove new lines? I see it more readable rather then on block of test

Reply:
That's what `yq` tool did that I use to replace version in yaml.

I can try to refactor to use multi-line `sed` if you wish.

Reply:
Oh, you dynamically update it...

Reply:
Maybe just read it from the toolchain file instead of hardcoding?

Suggestion from ChatGPT:
```
jobs:
  build:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
      # Checkout your repository

    - name: Extract Rust toolchain version
      id: rust_toolchain
      run: |
        TOOLCHAIN_VERSION=$(grep channel rust-toolchain.toml | awk '{print $3}' | tr -d '"')
        echo "TOOLCHAIN_VERSION=$TOOLCHAIN_VERSION" >> $GITHUB_ENV
        echo "::set-output name=version::$TOOLCHAIN_VERSION"
      # This command reads the rust-toolchain.toml, extracts the version, and sets it as an environment variable and step output

    # Use the extracted version in subsequent steps
    - name: Use Rust toolchain
      uses: actions-rs/toolchain@v1
      with:
        toolchain: ${{ steps.rust_toolchain.outputs.version }}
        override: true
```

I would use `$GITHUB_OUTPUT` though